cmd/copy.go

```diff
@@ -53,30 +66,32 @@ func Copy(cmd *cobra.Command, args []string) error {
 	return nil
 }
 
-func isLocalDirToRemoteCopy(args []string) bool {
-	if strings.Contains(args[0], "sess:") {
+func shouldCopyLocalDirToRemote(localPath1, remotePath2 string) bool {
+	if strings.Contains(localPath1, "sess:") {
 		return false
 	}
 
-	pathInfo, err := os.Stat(args[0])
+	pathInfo, err := os.Stat(localPath1)
 	if err != nil || pathInfo == nil {
 		return false
 	}
 
 	return pathInfo.IsDir()
 }
 
-func isRemoteDirToLocalCopy(args []string) bool {
-	if strings.Contains(args[1], "sess:") {
+func shouldCopyRemoteDirToLocal(remotePath1, localPath2 string) bool {
+	if !strings.Contains(remotePath1, "sess:") {
 		return false
 	}
 
-	pathInfo, err := os.Stat(args[1])
-	if err != nil || pathInfo == nil {
-		return false
-	}
+	sessFromDirpath := splitSessFromDirpath(remotePath1)
+	return isValidDirPath(sessFromDirpath)
+}
 
-	return pathInfo.IsDir()
+var regExValidDirpath = regexp.MustCompile(`^\/(?:[^\/]+\/)*[^\/]+$`)
```

Comment:
I suggest putting all the local variable declarations together at the top of the file instead of in-between functions